change/office-ui-fabric-react-2020-02-06-17-42-11-deprecation.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "comment": "Removing use of some deprecated things by replacing them.",
```

Comment:
Can you make this comment more useful? It goes into release notes. :)

What components are being updated"?

Reply:
Sure, I'll update the change file.